specification/authorization/resource-manager/Microsoft.Authorization/preview/2019-08-01-preview/authorization-UsageMetricsCalls.json

```diff
@@ -0,0 +1,105 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AuthorizationManagementClient",
+    "version": "2019-08-01-preview",
```

Comment:
>"version": "2019-08-01-preview" [](start = 4, length = 31)

This seems like a very old API version. Is there a reason for checking it in the swagger repo now?

Reply:
It's to fix s360 issues during the window to fix broken, this preview version is still in use and was flagged for not being documented